lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -143,13 +143,14 @@ int location_service_get_cell_location_nrf_cloud(
 	struct nrf_cloud_rest_context rest_ctx = {
 		.connect_socket = -1,
 		.keep_alive = false,
-		.timeout_ms = NRF_CLOUD_REST_TIMEOUT_NONE,
+		.timeout_ms = (params->timeout > 0) ? params->timeout * 1000 :
+						      NRF_CLOUD_REST_TIMEOUT_NONE,
```

Comment:
can you update the comment for the `timeout_ms` param: https://github.com/nrfconnect/sdk-nrf/blob/e08f3cf9a17f58358799af7984eb9f3a0fe1140e/include/net/nrf_cloud_rest.h#L68

Reply:
Done. We have now three places with this similar comment:
`nrf_cloud_rest_context.timeout_ms`
`rest_client_req_context.timeout_ms`
`REST_CLIENT_REQUEST_TIMEOUT`